contracts/extensions/VotingReputation.sol

```diff
@@ -232,6 +242,9 @@ contract VotingReputation is ColonyExtension, PatriciaTreeProofs {
 
     address target = getTarget(_altTarget);
     bytes4 action = getSig(_action);
+    if (action == bytes4(keccak256("moveFundsBetweenPots(uint256,uint256,uint256,uint256,uint256,uint256,address)"))){
+      require(false, "voting-rep-disallowed-function");
```

Comment:
Could be `require(action != ..., "voting...")`

Reply:
Wow, yeah, not sure what came over me here.